pymeasure/instruments/texio/texioPSW360L30.py

```diff
@@ -0,0 +1,71 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.keithley.keithley2260B import Keithley2260B
+
+import time
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class TexioPSW360L30(Keithley2260B):
+    r""" Represents the TEXIO PSW-360L30 Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the device only accepts
+    connections at port 2268, which cannot be configured otherwise.
+    example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
+
+    For a connection through USB on Linux, the kernel is going to create
+    a /dev/ttyACMX device automatically. The serial connection properties are
+    fixed at 9600–8-N-1.
+
+    The read termination for this interface is Line-Feed \n.
+
+    This driver inherits from the Keithley2260B one. All instructions
+    implemented in the Keithley 2260B driver are also available for the
+    TEXIO PSW-360L30 power supply.
+
+    The only addition is the "output" property that is just an alias for
+    the "enabled" property of the Keithley 2260B. Calling the output switch
+    "enabled" is confusing because it is not clear if the whole device is
+    enabled/disable or only the output.
+
+    .. code-block:: python
+
+        source = TexioPSW360L30("TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET")
+        source.voltage = 1
+        print(source.voltage)
+        print(source.current)
+        print(source.power)
+        print(source.applied)
+    """
+
+    def __init__(self, adapter, **kwargs):
+        Instrument.__init__(self,
+                            adapter=adapter,
+                            name="TEXIO PSW-360L30 Power Supply",
+                            read_termination="\n",
+                            **kwargs
+                            )
+
+    def check_errors(self):
+        """ Logs any system errors reported by the instrument.
+        """
+        code, message = self.error
+        while code != 0:
+            t = time.time()
+            log.info("TEXIO PSW-360L30 reported error: %d, %s" % (code, message))
+            code, message = self.error
+            if (time.time() - t) > 10:
+                log.warning("Timed out for TEXIO PSW-360L30 error retrieval.")
+
+    @property
+    def output(self):
+        """Same as "enabled" property"""
+        return self.enabled
+
+    @output.setter
+    def output(self, value):
+        """Same as "enabled" property"""
+        self.enabled = value
```

Comment:
You know what, I fully agree with you re: the bad naming for `enabled`. An even better one than `output` is `output_enabled`. :D 
I have a proposition:
* In the Keithley2260B implementation, rename `enabled` to `output_enabled`
* Adapt that in the `shutdown` method, too.
* Immediately below the new `output_enabled`, add this
```
    @property
    def enabled(self):
        warn("Deprecated property name "enabled", use the identical "output_enabled", instead.",
             FutureWarning)
        return self.enabled

    @output.setter
    def enabled(self, value):
        warn("Deprecated property name "enabled", use the identical "output_enabled", instead.",
             FutureWarning)
        self.enabled = value
```
* Add a note in the CHANGES.rst in the deprecated features section about the preferred name for Keithley2260B.enabled.
* Drop this here as it is now unnecessary :party:
What do you think?

Reply:
@bilderbuchi I fully agree with all your suggestions. I only have one question about the `shutdown` method. There are scenarios where you want to keep the power supply on, even if the software is not running.  How to deal with it?

Reply:
@bilderbuchi I do not see a deprecated features section in the CHANGES.rst file in my branch.

Reply:
> I fully agree with all your suggestions. I only have one question about the `shutdown` method. There are scenarios where you want to keep the power supply on, even if the software is not running. How to deal with it?

I understand that reason. Sometimes I want to close the program, but keep the source enabled. You do not need to call `shutdown`. It's only called automatically if you use the instrument in a context manager. Otherwise you have to call `shutdown` manually.


> I do not see a deprecated features section in the CHANGES.rst file in my branch.

Then simply add such a section.

Reply:
Ah, you missed this in the first bullet, I think: "In the Keithley2260B implementation,..." - those changes should go in that instrument's file, not the texio... :-P

Reply:
Yes, I misunderstood that. Now everything should fixed in commit 16a9c6fd88635f98a7d191e012ca0825f93a1cd9